app/screens/forgot_password/forgot_password.js

```diff
@@ -173,6 +173,7 @@ export default class ForgotPassword extends PureComponent {
 const style = StyleSheet.create({
     container: {
         flex: 1,
+        backgroundColor: 'white',
```

Comment:
why are we setting the background color to white? shouldn't this screen follow the OS sync theme dark/light?